configure.ac

```diff
@@ -460,8 +460,8 @@ AM_CONDITIONAL([xvmcompat], [test "x$xvmcompat" == "xyes"])
 # Try to detect the appropriate conf dir. Several systems have both /etc/default
 # and /etc/sysconfig but latter is always primary.
 AC_ARG_VAR(initconfdir, [directory for initscripts configuration])
-if test "x$initconfdir" = x && test "x$FENCE_VIRT" = "x1"; then
-	AC_CHECK_FILE(/etc/conf.d, [initconfdir='$(sysconfdir)/conf.d}'], [# Gentoo/Arch
+if test "x$initconfdir" = x; then
```

Comment:
This fails on BSD (see CI error). I'm not sure if there's an equivalent there.

Reply:
The fix makes it fail on lots of different archs on RHEL, e.g.:
https://ci.kronosnet.org/job/fence-agents/job/fence-agents-pipeline/job/PR-591/5/pipeline-console/?selected-node=538

Reply:
Removing the newline didnt make any difference, and also makes it harder to see that it's the closing statement.